src/shared/Atlassian.Bitbucket/BitbucketAuthentication.cs

```diff
@@ -88,7 +88,11 @@ public async Task<CredentialsPromptResult> GetCredentialsAsync(Uri targetUri, st
             // We need at least one mode!
             if (modes == AuthenticationModes.None)
             {
-                throw new ArgumentException(@$"Must specify at least one {nameof(AuthenticationModes)}", nameof(modes));
+                var format = @"Must specify at least one {0}";
```

Comment:
As I said in the previous commit, splitting up the format and message to use the 2 arg version of Trace2.WriteError() seems like a pain for many calls.

I don't have good visibility here to the `nameof(AuthenticationModes)` symbol here, so I can't say if this is useful split or not.  I don't think there's any PII here, but it might be a good use of a split if we want to group OAuth vs XYZ vs ABC schemes.

Reply:
I honestly just split anything that was interpolated. And yes, it was a pain - it took up a fair part of yesterday 😢 . Probably worth pairing on Monday so I fully understand when to split things vs. when not to so as not to use up more time spinning on it unnecessarily. I'll grab some time!